modules/nf-core/president/tests/main.nf.test

```diff
@@ -0,0 +1,104 @@
+import java.text.SimpleDateFormat
+
+nextflow_process {
+
+    name "Test Process PRESIDENT"
+    script "../main.nf"
+    process "PRESIDENT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "president"
+
+    test("president - nocompress") {
+
+        when {
+            process {
+                """             
+                input[0] = [[],file(params.test_data['sarscov2']['genome']['all_sites_fas'], checkIfExists: true)]
+                input[1] = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                input[2] = ""
+                """
+
+            }
+        }
+
+        then {
+            def firstReportEntry = path(process.out.report.get(0).get(1)).readLines()[1].tokenize("\t")
+            def firstReportAssert = ['sample1', 'all_sites.fas', 'genome.fasta', 'False', 'False', 'False', 'False', 'False', 'True', 'True', 'False', '27627', '2276', '0', '2276', '29903', '29829', '473', '0']
+            def date = new Date()
+            def sdf = new SimpleDateFormat("yyyy-MM-dd")
+            firstReportAssert.add(sdf.format(date))
+
+            assertAll{
```

Comment:
```suggestion
assertAll(
                { assert process.success },
                { assert process.out.fcz }
)                
```

assertAll syntax is as shown above

Reply:
honestly I don't understand it then. what is the advantage of wrapping them all into a assertAll block, but you have to put `assert` in front of every statement anyways?

Reply:
fixed

Reply:
thanks...assertAll(Closure... closures) ensures that all supplied closures do no throw exceptions. The number of failed closures is reported in the Exception message. This useful for efficient debugging of a set of test assertions from a single test run....basically to not fail the entire test workflow when encountered a failed assertion